mdn/yari
# use front-matter in index.html files

Fixes #643

To test:
1. `node content import --start-clean`
2. Check the generated files in `content/files/**/index.html`. No `index.yaml` file!
3. `yarn clean && yarn start`
4. Visit a page that has flaws. E.g. http://localhost:3000/en-US/docs/Web/CSS/background-position#_flaws
5. Check that clicking the lines in the flaws opens the right spot. 
6. Use the in-browser editor http://localhost:3000/en-US/_edit/Web/CSS/background-position and it should simply work. 

When you run `yarn start` you'll see something like this:
```
5:29:02 PM watchcontent.1 |  File watcher set up for /Users/peterbe/dev/MOZILLA/MDN/yari/content/files. Watching over 44,978 files in 44,960 folders.
```
Which is great because it's half as many files to watch.
